web/ui/react-app/src/pages/graph/ExpressionInput.tsx

```diff
@@ -19,6 +20,7 @@ interface ExpressionInputProps {
 
 interface ExpressionInputState {
   height: number | string;
+  showExpressionExplorer: boolean;
```

Comment:
Here and in some other places it's still using the "expression explorer" naming vs. "metrics explorer". (also in the modal dialog title)